src/data/code-samples/en/accessibility-group/swift-ui.md

```diff
@@ -0,0 +1,16 @@
+# Accessibility group - SwiftUI
+
+In SwiftUI, you can group multiple elements by applying the [`.accessibilityElement(children:)`](https://developer.apple.com/documentation/swiftui/view/accessibilityelement(children:)) modifier with the [`.combine`](https://developer.apple.com/documentation/swiftui/accessibilitychildbehavior/combine) option to a parent view. This approach consolidates the accessibility elements into a single group. Additionally, ensure that you provide an [`accessibilityLabel`](https://developer.apple.com/documentation/swiftui/view/accessibilitylabel(_:)-1d7jv) to describe the group as a whole.
+
+```swift
+@State var stockPrice = "$123.45"
+var body: some View {
+    HStack {
+        Text("Stock price:")
+        Text(stockPrice)
+    }
+    // Combine all text elements into a single accessibility element
+    .accessibilityElement(children: .combine)
+    .accessibilityLabel("The stock price is: \(stockPrice)")
```

Comment:
I think you don't need accessibilityLabel if the text is equal to the combined text of the elements. Maybe you can change the explanatory text to say something like: "You can optionally provide an `accessibilityLabel` to provide a custom description for the group`